src/main/app/feature-flags/flags.ts

```diff
@@ -2,6 +2,7 @@ export const BETA_ADD = 'idam-user-dashboard--beta-add';
 export const BETA_EDIT = 'idam-user-dashboard--beta-edit';
 export const BETA_SUSPEND = 'idam-user-dashboard--beta-suspend';
 export const BETA_DELETE = 'idam-user-dashboard--beta-delete';
+export const BETA_SSO = 'idam-user-dashboard--beta-sso';
```

Comment:
This should be BETA_REMOVE_SSO. This name of the string  should be 'idam-user-dashboard--beta-remove-sso'

Reply:
fixed